core/store/models/log_consumption.go

```diff
@@ -7,44 +7,22 @@ import (
 	"github.com/smartcontractkit/chainlink/core/eth"
 )
 
-// LogConsumerTypeJob - LogConsumptions with this type were consumed by a job
-const LogConsumerTypeJob = "job"
-
-// LogConsumerTypes holds the list of valid consumer types
-var LogConsumerTypes = [1]string{LogConsumerTypeJob}
-
-// A LogConsumption is a unique record indicating that a particular consumer has
+// A LogConsumption is a unique record indicating that a particular job has
 // already consumed a particular log. This record can be used to prevent consumers
 // from re-processing duplicate logs
 type LogConsumption struct {
-	ID           *ID
-	BlockHash    common.Hash
-	LogIndex     uint
-	ConsumerType string
-	ConsumerID   *ID
-	CreatedAt    time.Time
-}
-
-// A LogConsumer has a type and ID, and uniquely identifies a LogListener
-type LogConsumer struct {
-	Type string
-	ID   *ID
+	ID         uint
+	BlockHash  common.Hash
+	LogIndex   uint
+	ConsumerID *ID
```

Comment:
If it's not null why is it a pointer here?

Reply:
Pointers as struct members actually use more space than a regular field @RyanRHall. i.e. space for the pointer and the record. 

Using non pointer IDs with gorm seems to be a bit of a headache atm and will require some refactoring of the ID struct which might be a bit out of scope for this PR.

Reply:
TIL. Thanks @j16r